components/tor/tor_launcher_factory.h

```diff
@@ -96,7 +101,11 @@ class TorLauncherFactory : public tor::TorControl::Delegate {
 
   scoped_refptr<base::SequencedTaskRunner> file_task_runner_;
 
-  std::unique_ptr<tor::TorControl> control_;
+  std::unique_ptr<tor::TorControl, base::OnTaskRunnerDeleter> control_;
+
+  tor::TorFileWatcher* tor_file_watcher_;
```

Comment:
I may be wrong, but we don't actually need this pointer, since it is always a single-shot self-deleting object

Reply:
fixed in 47093e7b5b5e8e93172696c68a52086fb3a1aaf6